sdk/src/main/scala/io/horizen/account/state/nativescdata/forgerstakev2/StakeStorage.scala

```diff
@@ -140,24 +140,105 @@ object StakeStorage {
     forgerHistory.updateOrAddCheckpoint(view, forgerHistorySize, epochNumber, subtractStake)
   }
 
-  def getAllForgerStakes(view: BaseAccountStateView): Seq[ForgerStakeData] = {
+  def getForgingStakes(view: BaseAccountStateView): Seq[ForgingStakeInfo] = {
+    val listOfForgerKeys = ForgerMap.getForgerKeys(view)
+    listOfForgerKeys.map { forgerKey =>
+      val forger = ForgerMap.getForger(view, forgerKey)
+      val amount = ForgerStakeHistory(forgerKey).getLatestAmount(view)
+      ForgingStakeInfo(forger.forgerPublicKeys.blockSignPublicKey, forger.forgerPublicKeys.vrfPublicKey, ZenWeiConverter.convertWeiToZennies(amount))
+    }
+  }
+
+  def getAllForgerStakes(view: BaseAccountStateView): Seq[AccountForgingStakeInfo] = {
     val listOfForgerKeys = ForgerMap.getForgerKeys(view)
     listOfForgerKeys.flatMap { forgerKey =>
       val forger = ForgerMap.getForger(view, forgerKey)
       val delegatorList = DelegatorList(forgerKey)
       val delegatorSize = delegatorList.getSize(view)
-      val listOfStakes: ListBuffer[ForgerStakeData] = ListBuffer()
+      val listOfStakes: ListBuffer[AccountForgingStakeInfo] = ListBuffer()
       for (idx <- 0 until delegatorSize) {
         val delegator = delegatorList.getDelegatorAt(view, idx)
         val stakeHistory = StakeHistory(forgerKey, delegator)
         val amount = stakeHistory.getLatestAmount(view)
-        if (amount.signum() == 1)
-          listOfStakes.append(ForgerStakeData(forger.forgerPublicKeys, new AddressProposition(delegator), amount))
+        if (amount.signum() == 1) {
+          val stakeId = Blake2b256.hash(Bytes.concat(forgerKey.bytes, delegator.toBytes))
```

Comment:
Refactored this method so that it returns only data that is known. empty value for stakeId is added later in state view to keep the same datastructure for existing API endpoint